clash-lib/src/Clash/Core/Evaluator/Models.hs

```diff
@@ -0,0 +1,258 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Clash.Core.Evaluator.Models where
```

Comment:
Could you fill me in on the name "Models"? I suspect that this is jargon I should know, but I don't..

Reply:
Discussed offline: For now the module is called `Models` (because it models partial evaluation), but when the old evaluator is gone the module can be called `Types`